kubelet/datadog_checks/kubelet/common.py

```diff
@@ -38,6 +38,14 @@ def tags_for_docker(cid, cardinality):
     return tagger.tag('container_id://%s' % cid, cardinality) or []
 
 
+def tags_for_container(prefixed_cid, cardinality):
```

Comment:
I'd prefer not having both `tags_for_docker` and `tags_for_container` - we should rather have only one function `tags_for_container` with the ability to choose if we want to add the prefix via a bool argument

Reply:
You're right that's better